docs/docs-content/clusters/edge/local-ui/cluster-management/link-hosts.md

```diff
@@ -0,0 +1,172 @@
+---
+sidebar_label: "Link Hosts"
+title: "Link Hosts"
+description: "Instructions for linking hosts to prepare for multi-node cluster creation."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["edge"]
+---
+
+To create a multi-node cluster with hosts provisioned in the airgap installation mode, the hosts must first be able to
+identify and securely communicate with each other. By default, hosts that are provisioned in the airgap installation
+mode are not aware of each other even if they are on the same network, and they do not have the credentials to
+communicate with each other securely.
+
+![A diagram of the order of operations for linking hosts.](/clusters_edge_localui_cluster-mgmt_link-hosts.webp)
+
+Host linking provides the hosts with the necessary network and security infrastructure to form a cluster together. In a
+group of linked hosts, hosts can broadcast information to all its peers. Every group has a leader node, which provides
+tokens with its IP and a One-Time Password (OTP) credentials that you can use to link other nodes.
+
+## Link Hosts
+
+Linked hosts will sync their status and uploaded content, including images for the Palette agent and provider images,
+with each other. In a group of linked hosts that have not formed a cluster, only the leader node has access to
+functionality such as to change host settings, upload content, and create clusters. Once a cluster is created, all
+control plane nodes will be considered leader nodes and only the control plane nodes have access to cluster and host
+management features.
```

Comment:
Hmmm....I think this concept of all control-plane nodes being leader nodes is a bit confusing.  I'm not sure I understand what being a leader means here. I drew out a visual to help me understand. Is a leader here in the context of LocaUI's scope? Maybe if I understood what a leader does/is responsible for it would help me? 😅 

![image](https://github.com/user-attachments/assets/fb986b55-7c22-495f-83eb-e5ae1bdf43d3)

Reply:
Abhinav noted this in his review that all control plane nodes are considered leaders once a cluster is formed so I added that in. Now that i think about it, it is a bit confusing. 

I think instead of saying that, it's probably better if we decouple the control plane concepts with the leader concepts. I think what Abhi said was more in the context of "all the control plane nodes have access to cluster management features", which in a group of linked nodes, only leader nodes have access to. But if you delete that cluster, there will still only be one leader to which all the other nodes are going to be linked. I will confirm this with him